### PR TITLE
Variant mobile optimisation test1 100%, Remove AB-test

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What are you doing in this PR?
First mobile optimisation test success, make variant the control.

1. make the variant = control (supporterPlusMobileTest1)
2. remove ab-test 'supporterPlusMobileTest1'
3. cleanup default component parameters

[**Trello Card**](https://trello.com/c/VHzUhykn/1271-end-supporterplusmobiletest1-a-b-test-and-put-the-variant-live)

## Is this an AB test?
- [x] Yes

## Original PR
https://github.com/guardian/support-frontend/pull/4791

### CODE REVIEW

**Comment by paul-daniel-dempsey** [comment]
[End supporterPlusMobileTest1 A/B test and put the variant live](https://trello.com/c/VHzUhykn/1271-end-supporterplusmobiletest1-a-b-test-and-put-the-variant-live)

File: support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListContainer.tsx [hunk 1]
```diff
@@ -123,9 +116,6 @@ export function CheckoutBenefitsListContainer({
 		),
 		handleButtonClick,
 		countryGroupId,
-		rowSpacingNarrow: useOptimisedMobileLayout,
-		cssOverrides: useOptimisedMobileLayout
-			? optimisedLayoutOverrides
-			: undefined,
+		cssOverrides: layoutOverrides,
```
Comment (GHaberis) [inline support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListContainer.tsx:119]: So we need to pass these style rules as `cssOverrides`? I can see the `cssOverrides` were added when set-up the first Mobile Optimisation test [here](https://github.com/guardian/support-frontend/commit/953c9e89237383d1733d4373d451c6cc3a1ad7a7#diff-9f7effdc73d8054d2d6b5b25467864c2e6c12e4de2c8a26244ba948bad8e2d23R126). Can these style rules just be moved to the default rules of the component?

Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListContainer.tsx]: Agree these overrides simply are not needed. Have removed css overrides from containers->  benefitslist, pricecards and paymentmethods (all ones directly added within the original PR). The styles have been placed into their components. Completed, thanks.

File: support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx [hunk 2]
```diff
@@ -194,38 +189,23 @@ export function SupporterPlusLandingPage({
...
 			</CheckoutHeading>
-			<Container
-				sideBorders
-				cssOverrides={
-					optimisedMobileLayout
-						? darkBackgroundContainerMobile
-						: backgroundContainer
-				}
-			>
+			<Container sideBorders cssOverrides={darkBackgroundContainerMobile}>
```
Comment (marialani) [inline support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx:192]: Is the `backgroundContainer` css being used elsewhere? Or can that be stripped out now?

File: support-frontend/assets/components/checkoutBenefits/checkoutBenefitsList.tsx [hunk 3]
```diff
@@ -5,18 +5,26 @@ import {
...
-import type { CSSOverridable } from 'helpers/types/cssOverrideable';
 import type { CheckListData } from './checkoutBenefitsListData';
 
 const container = css`
 	${textSans.medium({ lineHeight: 'tight' })};
+	${until.tablet} {
+		tr {
+			border-bottom-width: 6px;
+		}
+	}
```
[the remark this answers is not in the document]
Comment (GHaberis) [reply support-frontend/assets/components/checkoutBenefits/checkoutBenefitsList.tsx:20]: Yes and no, the rule was present, it's just it was defined elsewhere. It was originally being passed as `cssOverride`, the override was only applied to the `variant`. Now the `variant` is becoming the default we're applying the rule as default without an override.

[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/checkoutBenefits/checkoutBenefitsList.tsx:20]: Yes wasin checkoutBenefitListContainer that was moved into checkoutBenefitList

File: support-frontend/assets/components/checkoutBenefits/checkoutBenefitsList.tsx [hunk 4]
```diff
@@ -64,29 +72,29 @@ const hr = (margin: string) => css`
...
-	rowSpacingNarrow,
-	cssOverrides,
 }: CheckoutBenefitsListProps): JSX.Element {
 	return (
-		<div css={[container, cssOverrides]}>
+		<div css={container}>
 			<h2 css={heading}>{title}</h2>
 			<hr css={hr(`${space[4]}px 0`)} />
-			<table css={table(rowSpacingNarrow ? `6` : `10`)}>
+			<table css={table(`6`)}>
```
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/checkoutBenefits/checkoutBenefitsList.tsx]: Yes, will try to spot these in future. Changed with commit 'table spacing used once, hardcoded'. Thanks.

File: support-frontend/assets/components/checkoutBenefits/checkoutBenefitsList.tsx [hunk 5]
```diff
@@ -5,18 +5,26 @@ import {
...
+			border-bottom-width: 6px;
+		}
+	}
 `;
 
 const heading = css`
-	${headline.small({ fontWeight: 'bold', lineHeight: 'tight' })};
+	${headline.xsmall({ fontWeight: 'bold' })}
+	${from.tablet} {
+		${headline.small({ fontWeight: 'bold', lineHeight: 'tight' })};
```
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/checkoutBenefits/checkoutBenefitsList.tsx]: Yes have replaced it here with font-size: 28px; + line-height: 115%; Thanks.

File: support-frontend/assets/components/paymentMethodSelector/paymentMethodSelector.tsx [hunk 6]
```diff
@@ -23,14 +22,12 @@ const container = css`
 
 const header = css`
 	margin-bottom: ${space[3]}px;
-	${headline.small({ fontWeight: 'bold' })};
-`;
-
-const hidesecuretransactionindicator = css`
-	${until.tablet} {
-		display: none;
+	${headline.xsmall({ fontWeight: 'bold' })}
```
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/paymentMethodSelector/paymentMethodSelector.tsx]: Yes have corrected and replaced following line27 with font-size: 28px;  Thanks.

File: support-frontend/assets/components/paymentMethodSelector/paymentMethodSelector.tsx [hunk 7]
```diff
@@ -23,14 +22,12 @@ const container = css`
...
+	${from.tablet} {
+		${headline.small({ fontWeight: 'bold' })};
 	}
 `;
+
 const showsecuretransactionindicator = css`
```
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/paymentMethodSelector/paymentMethodSelector.tsx]: Yes have renamed as suggested, including 'Overrides' in name would guess is more appropriate. Thanks

File: support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx (hunk 2 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx:188]: Yes have merged them into `darkBackgroundContainerMobile`. Thanks

**Review by marialani — APPROVED** [review]
LGTM! 👍

**Pull request merged**